### DISCUSSION TITLE
What quests did I solve in Austria?

### DISCUSSION THREAD

**Discussion by AndreasHoffmann2** [streetcomplete/StreetComplete#3808] [Q&A] [answered]
Hi,

I started using StreetComplete a few weeks ago. So far, I solved quests in Germany, France and Italy.
But today, Austria showed up in my country-statistics suddenly. With 30 points.

How can this be?

I used [overpass](http://overpass-turbo.eu/?Q=%0A%5Bout%3Ajson%5D%5Btimeout%3A240%5D%3B%0A%0A(%20%20%0A%20%20way(%7B%7Bbbox%7D%7D)(user%3A%22Epaminaidos%22)%3B%0A%20%20node(%7B%7Bbbox%7D%7D)(user%3A%22Epaminaidos%22)%3B%0A%20%20relation(%7B%7Bbbox%7D%7D)(user%3A%22Epaminaidos%22)%3B%0A%20%20%0A)%3B%0A%2F%2F%20print%20results%0Aout%20body%3B%0A%3E%3B%0Aout%20skel%20qt%3B&C=47.17851;13.40881;7) and tried to find the edits, but found nothing. But it is very possible that I did not create the correct query.

How can I find out, why StreetComplete is showing these 30 points?

I don't want to insert any invalid data into OSM. And having the feeling that there are 30 edits that I did not intend to do bothers me.

**Comment by FloEdelmann** [comment]
I created a [filter](https://osmcha.org/filters?filters=%7B%22date__gte%22%3A%5B%7B%22label%22%3A%222022-01-01%22%2C%22value%22%3A%222022-01-01%22%7D%5D%2C%22editor%22%3A%5B%7B%22label%22%3A%22StreetComplete%22%2C%22value%22%3A%22StreetComplete%22%7D%5D%2C%22users%22%3A%5B%7B%22label%22%3A%22Epaminaidos%22%2C%22value%22%3A%22Epaminaidos%22%7D%5D%2C%22geometry%22%3A%5B%7B%22label%22%3A%7B%22coordinates%22%3A%5B%5B%5B12.95761%2C47.629996%5D%2C%5B12.584491%2C48.152129%5D%2C%5B13.833628%2C48.808087%5D%2C%5B14.482531%2C48.733244%5D%2C%5B15.017875%2C49.106344%5D%2C%5B17.01325%2C48.786714%5D%2C%5B17.256589%2C47.651856%5D%2C%5B16.640131%2C46.980987%5D%2C%5B16.121009%2C46.592214%5D%2C%5B14.563643%2C46.279173%5D%2C%5B12.292485%2C46.614505%5D%2C%5B10.92979%2C46.65906%5D%2C%5B9.940214%2C46.792505%5D%2C%5B9.291311%2C47.069455%5D%2C%5B9.485982%2C47.651856%5D%2C%5B10.475558%2C47.651856%5D%2C%5B11.189351%2C47.542461%5D%2C%5B12.292485%2C47.81552%5D%2C%5B12.95761%2C47.629996%5D%5D%5D%2C%22type%22%3A%22Polygon%22%7D%2C%22value%22%3A%7B%22coordinates%22%3A%5B%5B%5B12.95761%2C47.629996%5D%2C%5B12.584491%2C48.152129%5D%2C%5B13.833628%2C48.808087%5D%2C%5B14.482531%2C48.733244%5D%2C%5B15.017875%2C49.106344%5D%2C%5B17.01325%2C48.786714%5D%2C%5B17.256589%2C47.651856%5D%2C%5B16.640131%2C46.980987%5D%2C%5B16.121009%2C46.592214%5D%2C%5B14.563643%2C46.279173%5D%2C%5B12.292485%2C46.614505%5D%2C%5B10.92979%2C46.65906%5D%2C%5B9.940214%2C46.792505%5D%2C%5B9.291311%2C47.069455%5D%2C%5B9.485982%2C47.651856%5D%2C%5B10.475558%2C47.651856%5D%2C%5B11.189351%2C47.542461%5D%2C%5B12.292485%2C47.81552%5D%2C%5B12.95761%2C47.629996%5D%5D%5D%2C%22type%22%3A%22Polygon%22%7D%7D%5D%7D) for these circumstances in OsmCha:
* your user name
* Streetcomplete editor
* since Janurary 1st 2022
* in (rough) boundaries of Austria

It yields only one changeset: https://www.openstreetmap.org/changeset/117743032
That changeset has a large bounding box which indeed overlaps Austria, but none of the changes are anywhere near the country.

**Comment by westnordost** [comment, marked as answer]
For the statistics, only the center of the bounding box of the changeset is considered. I didn't calculate, but probably the geographical center of that bounding box in in Austria.

Note that the outline of a bounding box rectangle on the surface of a sphere has a trapezoid shape.
